### PR TITLE
Use a session w/o a WL policy for schema changes

### PR DESCRIPTION
**Pull request by ptnapoleon**
@knifewine to review

### CODE REVIEW

**Comment by ptnapoleon** [comment]
Fixes 11951

File: user_functions_test.py [hunk 1]
```diff
@@ -46,24 +43,27 @@ def test_migration(self):
         node3 = cluster.nodelist()[2]
         time.sleep(0.2)
 
-        session1 = self.patient_exclusive_cql_connection(node1)
-        session2 = self.patient_exclusive_cql_connection(node2)
-        session3 = self.patient_exclusive_cql_connection(node3)
-        self.create_ks(session1, 'ks', 1)
-        session2.execute("use ks")
-        session3.execute("use ks")
+        # The later three sessions use a whitelist policy, and then don't wait for schema agreement
```
Comment (mambocab) [inline user_functions_test.py]: `latter`?

These variable names should document either their use, or the fact that they have/don't have a whitelist policy.

Comment (ptnapoleon) [inline user_functions_test.py]: Debatably `latter`, yes.

Now, we all know that the `exclusive` functions have a whitelist policy and the non-`exclusive` functions do not. How would you prefer I rename these? Anything more verbose than `session_to_node1`, I'm -1 on

Comment (mambocab) [inline user_functions_test.py]: > Now, we all know that the exclusive functions have a whitelist policy

The variable names could still document the purposes of the different sessions, or help us in the case where someone editing/reviewing doesn't read the comments. My thought was maybe `schema_wait_session` vs. `nodeN_query_session`, or `no_whitelist_session` vs. nodeN_session`.

**Comment by ptnapoleon** [comment]
Changed made.

**Comment by mambocab** [comment]
I'm +1 if you've run locally. @knifewine?

**Comment by knifewine** [comment]
belated +1

**Pull request merged**